### DISCUSSION TITLE
MAA运行时被 windows defender 判断为木马程序

### DISCUSSION THREAD

**Discussion by HaohuaFeng** [MaaAssistantArknights/MaaAssistantArknights#6708] [Q&A] [answered]
这几天更新完后每次启动就被windows defender判断为恶意程序直接隔离和删除掉。笔记本和台式都出现这样情况。
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/47614849/beffe73e-50e1-471c-9fea-721dbc5b3afb)

目前用的是是内测版的MAA：
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/47614849/d627e9e2-7e21-4757-979d-6d7a95584c6e)

**Comment by Constrat** [comment, marked as answer]
Duplicate of #6662
